src/cachers/redis.js

```diff
@@ -39,20 +39,36 @@ class RedisCacher extends BaseCacher {
 	 */
 	init(broker) {
 		super.init(broker);
-
-		let Redis;
+		let Redis, Redlock;
 		try {
 			Redis = require("ioredis");
 		} catch (err) {
 			/* istanbul ignore next */
 			this.broker.fatal("The 'ioredis' package is missing. Please install it with 'npm install ioredis --save' command.", err, true);
 		}
+		try {
+			Redlock = require('redlock');
+		} catch (err) {
+			/* istanbul ignore next */
+			this.broker.fatal("The 'redlock' package is missing. Please install it with 'npm install redlock --save' command.", err, true);
+		}
 
 		/**
 		 * ioredis client instance
 		 * @memberof RedisCacher
 		 */
 		this.client = new Redis(this.opts.redis);
+		let redlockClients = (this.opts.redlock ? this.opts.redlock.clients : null) || [this.client]
+		this.redlock = new Redlock(
+			redlockClients,
+			_.omit(this.opts.redlock, ['clients'])
+		);
+		this.redlockNonBlocking = new Redlock(
+			redlockClients,
+			{
+				retryCount: 0
```

Comment:
This setting would be good for everyone (I don't know)? If not, would be good to expose to `this.opts.redlock.retryCount`

Reply:
Yes, this setting would be good for everyone. There are two kinds of redlock clients internal. 
1. `redlock` : will take the `this.opts.redlock` as options, used for `cacher.lock(key)` method.
2. `redlockNonBlocking` will take `{ retryCount: 0 }` as options, used for `cacher.tryLock(key)` method, which will not block the request.